### PR TITLE
Overhaul the frame selector UI

### PR DESCRIPTION
**Pull request by eggrobin**
Allow one-click switching between all reference frames; a pinning mechanism allows for a reduced set of bodies to be displayed, making the UI smaller; the purpose of various reference frames is indicated in tooltips to help newer players.

Chinese translations to be added in a future pull request.

### CODE REVIEW

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 1]
```diff
@@ -42,12 +43,12 @@ public enum FrameType {
     frame_type = FrameType.BODY_CENTRED_NON_ROTATING;
     selected_celestial = FlightGlobals.GetHomeBody();
     is_freshly_constructed_ = true;
-
+
```
Comment (pleroy) [inline ksp_plugin_adapter/reference_frame_selector.cs:46]: Space at eol.

File: ksp_plugin_adapter/reference_frame_selector.cs [hunk 2]
```diff
@@ -375,85 +369,179 @@ public enum FrameType {
...
+          expanded_[celestial] = !expanded_[celestial];
         }
-        if (UnityEngine.GUILayout.Button(button_text, GUILayoutWidth(offset))) {
+      }
+      UnityEngine.GUILayout.Label(celestial.name);
+      UnityEngine.GUILayout.FlexibleSpace();
+      if (celestial.is_root()) {
+        UnityEngine.GUILayout.Label("Pin");
+      } else {
+        if (UnityEngine.GUILayout.Toggle(pinned_[celestial], "") !=
```
Comment (pleroy) [inline ksp_plugin_adapter/reference_frame_selector.cs:434]: `else if`

File: ksp_plugin_adapter/style.cs [hunk 3]
```diff
@@ -6,6 +6,25 @@ internal static class Style {
...
+    var style = new UnityEngine.GUIStyle(UnityEngine.GUI.skin.button);
+    style.active.textColor = ultra_cool_grey_;
+    style.hover.textColor = ultra_cool_grey_;
+    style.normal.textColor = ultra_cool_grey_;
+    return style;
+  }
+
+  public static UnityEngine.GUIStyle LitToggleButton() {
+    var style = new UnityEngine.GUIStyle(UnityEngine.GUI.skin.button);
+    style.onActive.textColor = XKCDColors.Green;
```
Comment (pleroy) [inline ksp_plugin_adapter/style.cs:19]: Find a plant.

Comment (eggrobin) [reply ksp_plugin_adapter/style.cs:19]: Spearmint.

**Pull request merged**